CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Next Release
 
+### Features
+
+- [In Progress: 17 Mar 2022] Messaging users to remind them to fill out drug stock report each month
+
+### Changes
+
+- [In Progress: 24 Mar 2022] Add stronger app update reminders
```

Comment:
@msasikanth is this an automated pr?

Reply:
No, was anything wrong with this change?

Reply:
oh no no. Was just curious